### PR TITLE
#690 - CredsConfigYaml created

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #690 
I've created `CredsConfigYaml` and used it in `api.artifactory` package classes

### CODE REVIEW

File: src/test/java/com/artipie/CredsConfigYaml.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    /**
+     * Adds users to credentials config.
+     * @param usernames Names to add
+     * @return Itself
+     */
+    public CredsConfigYaml withUsers(final String... usernames) {
+        for (final String name : usernames) {
+            this.builder = this.builder.add(
+                name, Yaml.createYamlMappingBuilder().add("pass", "pain:123").build()
```
Comment (olegmoz) [inline src/test/java/com/artipie/CredsConfigYaml.java:62]: @olenagerasimova I am pretty sure that it should be `plain:123`, not `pain`

File: src/test/java/com/artipie/CredsConfigYaml.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    /**
+     * Adds user with groups to credentials config.
+     * @param username Name of the user
+     * @param groups Groups list
+     * @return Itself
+     */
+    public CredsConfigYaml withUserAndGroups(final String username, final List<String> groups) {
+        YamlMappingBuilder user = Yaml.createYamlMappingBuilder().add("pass", "pain:123");
```
Comment (olegmoz) [inline src/test/java/com/artipie/CredsConfigYaml.java:75]: @olenagerasimova same typo

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see two comments

**Comment by olenagerasimova** [comment]
@olegmoz thanks, fixed

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#691` is not assigned, can't get performer

**Pull request merged**